.gitignore

```diff
@@ -37,3 +37,5 @@ loadtest_results.txt
 # nitro-replayer
 nitro-replayer/target
 nitro-replayer/Cargo.lock
+# nitro mac artifact
+x/nitro/replay/libnitro_replayer.dylib
```

Comment:
if we're adding this to gitignore, whys the file shown as modified below? shouldn't we remove it?